cafebabel/articles/tags/views.py

```diff
@@ -46,7 +47,13 @@ def edit(slug):
     tag = Tag.objects.get_or_404(slug=slug, language=current_language())
 
     if request.method == 'POST':
-        tag.save_from_request(request)
+        try:
+            tag.save_from_request(request)
+        except ValidationError as e:
+            # TODO: see https://github.com/cafebabel/cafebabel.com/issues/187
+            flash('There was an error in your tag submission:')
+            flash(str(e))
+            return redirect(url_for('tags.edit', slug=slug))
         flash('Your tag was successfully saved.')
```

Comment:
Oh, we're missing a category here too

Reply:
Let's keep the default by default/success?